src/components/Card/Card.scss

```diff
@@ -45,5 +64,14 @@
 .footer {
   display: flex;
   justify-content: flex-end;
-  padding: 0 spacing(loose) spacing(loose);
+  // padding: 0 spacing(loose) spacing(loose);
+}
+
+.cardTitle {
+  font-size: 24px;
+  font-family: OpenSans-Semibold;
```

Comment:
@HiirenP I am not sure if font-family is needed here, if yes then its better to add that in delicious theme itself in a common class, because it might be needed in many more places